lib/libc/string/Makefile.inc

```diff
@@ -28,12 +28,13 @@ MISRCS+=bcmp.c bcopy.c bzero.c explicit_bzero.c \
 	wmemcmp.c \
 	wmemcpy.c wmemmove.c wmempcpy.c wmemset.c
 
+SYM_MAPS+=	${LIBC_SRCTOP}/string/Symbol.map
+
 .if ${MACHINE_CPU:Mcheri} && !${MACHINE_ABI:Mpurecap}
 MISRCS+=memcpy_c.c memmove_c.c memset_c.c
+SYM_MAPS+=	${LIBC_SRCTOP}/string/Symbol_cheri.map
```

Comment:
Eh, that's not a great name given they apply to hybrid only. Also not sure of the merits of having a separate file vs just using the preprocessor.

Reply:
Yeah, I had played with having this use #ifdef before I had started passing CFLAGS and so it wasn't working.  I kind of liked relying on the same set of conditionals from the Makefile without having to spell it twice in #ifdef.  I could rename this to Symbol_cheri_hybrid.map if that is better?